pro-duck-store/src/components/Drawer/drawer.tsx

```diff
@@ -0,0 +1,105 @@
+import React from 'react';
+import clsx from 'clsx';
+import {createStyles, makeStyles, Theme} from '@material-ui/core/styles';
+import Drawer from '@material-ui/core/Drawer';
+import CssBaseline from '@material-ui/core/CssBaseline';
+import IconButton from '@material-ui/core/IconButton';
+import ChevronLeftIcon from '@material-ui/icons/ChevronLeft';
+import ChevronRightIcon from '@material-ui/icons/ChevronRight';
+import DrawerList from "./drawerList";
+
+
+
+const drawerWidth = 280;
+
+const useStyles = makeStyles((theme: Theme) =>
```

Comment:
You should extract this in a separate folder and import it from there. Something like `drawer.styles.tsx`